WiseUnpacker/SabreTools.Serialization.Deserializers/WiseSectionHeader.cs

```diff
@@ -63,6 +60,7 @@ public class WiseSectionHeader : BaseBinaryDeserializer<SectionHeader>
         private static SectionHeader? ParseWiseSectionHeader(Stream data, long initialOffset)
         {
             var header = new SectionHeader();
+            int localWisOffset = -1;
```

Comment:
Since you're introducing this variable, please check with `header.Version` that the value for the offset is `>= initialOffset`. This will reduce potential issues later.